packages/grid/_modules_/grid/models/gridSlotsComponent.ts

```diff
@@ -26,6 +26,9 @@ import { GridPanel, GridPanelProps } from '../components/panel/GridPanel';
 import { GridApiRefComponentsProperty } from './api/gridComponentsApi';
 import { GridIconSlotsComponent } from './gridIconSlotsComponent';
 import { GridBaseComponentProps } from './params/gridBaseComponentProps';
+import { GridToolbarContainerProps } from '../components/containers/GridToolbarContainer';
+import { GridFooterContainerProps } from '../components/containers/GridFooterContainer';
+import { GridOverlayProps } from '../components/containers/GridOverlay';
 
 export type ColumnMenuProps = GridBaseComponentProps & GridColumnMenuProps;
```

Comment:
you need to update this one as well. But you need to keep `GridColumnMenuProps`

Reply:
Fixed